main/src/ca/uwaterloo/flix/language/ast/KindedAst.scala

```diff
@@ -168,7 +168,7 @@ object KindedAst {
 
     case class InvokeConstructor(constructor: Constructor[_], args: List[KindedAst.Expression], loc: SourceLocation) extends KindedAst.Expression
 
-    case class InvokeMethod(method: Method, exp: KindedAst.Expression, args: List[KindedAst.Expression], loc: SourceLocation) extends KindedAst.Expression
+    case class InvokeMethod(method: Method, thisTpe: java.lang.Class[_], exp: KindedAst.Expression, args: List[KindedAst.Expression], loc: SourceLocation) extends KindedAst.Expression
```

Comment:
Could we rename `thisTpe` to `clazz` or `receiverClass`? (Mostly because it is not a Flix "type").

Reply:
Done